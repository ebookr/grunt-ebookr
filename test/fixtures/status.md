# Testing

<status state="1" />

## More testing

<status state="0" />