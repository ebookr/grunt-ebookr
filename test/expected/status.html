<h1 id="testing">Testing</h1>
<p>Status: <a href="http://test.com/#1">bar (1)</a></p>
<h2 id="more-testing">More testing</h2>
<p>Status: <a href="http://test.com/#0">foo (0)</a></p>
